Cold-Gas-Thruster/SPEXformat.cls

```diff
@@ -0,0 +1,4662 @@
+%% SPEXformat.cls 2017/2/24 version V0.0.1 based on
```

Comment:
This file can also be removed. In future versions, it won't even be included in the template. :)